tests/test_connector.py

```diff
@@ -56,7 +56,15 @@ def unix_server(loop, unix_sockname):
         runners.append(runner)
         await runner.setup()
         site = web.UnixSite(runner, unix_sockname)
-        await site.start()
+        try:
+            await site.start()
+        except OSError as exc:
+            if str(exc) == "AF_UNIX path too long":
+                if sys.platform == "linux":
+                    raise
+                pytest.skip("Some UNIXes has too strict limitation "
```

Comment:
Actually all of them. It just happens that the default tmp path prefixed there under macOS is longer (`/private/tmp/...` vs. `/tmp/...`). I think, the right way to fix it is to substitute the temp dir.
It should be possible to do `export TMPDIR=/tmp` in CI. Try it.


Ref: https://unix.stackexchange.com/a/367012/27133

Reply:
Also, in this case, it should be probably `xfail()` instead of `skip()` if you still want to keep it. But I don't like the idea of skipping tests when it's possible to address this properly.

Reply:
xfail sounds good!